internal/component/database_observability/mysql/collector/schema_table.go

```diff
@@ -241,27 +261,125 @@ func (c *SchemaTable) extractSchema(ctx context.Context) error {
 		var collationConnection string
 		if table.tableType == "BASE TABLE" {
 			if err = row.Scan(&tableName, &createStmt); err != nil {
-				level.Error(c.logger).Log("msg", "failed to scan create table", "table", table.tableName, "err", err)
+				level.Error(c.logger).Log("msg", "failed to scan create table", "schema", table.schema, "table", table.tableName, "err", err)
 				break
 			}
 		} else if table.tableType == "VIEW" {
 			if err = row.Scan(&tableName, &createStmt, &characterSetClient, &collationConnection); err != nil {
-				level.Error(c.logger).Log("msg", "failed to scan create view", "table", table.tableName, "err", err)
+				level.Error(c.logger).Log("msg", "failed to scan create view", "schema", table.schema, "table", table.tableName, "err", err)
 				break
 			}
 		}
 
 		table.createStmt = createStmt
 		c.cache.Add(cacheKey, table)
 
+		spec, err := c.parseTableSpec(table.schema, table.tableName, createStmt)
+		if err != nil {
```

Comment:
Do we not want to report this error in some way?

Reply:
Done!